jarviz-lib/pom.xml

```diff
@@ -231,6 +231,13 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
Without realizing, I've even fixed this
https://github.com/ExpediaGroup/jarviz/blob/master/pom.xml#L128-L134

Reply:
And this is already done, too.